modules/virt-configuring-vm-disk-sharing.adoc

```diff
@@ -10,6 +10,14 @@ You can configure block volumes so that multiple virtual machines (VMs) can shar
 
 The application running on the guest operating system determines the storage option you must configure for the VM. A disk of type `disk` exposes the volume as an ordinary disk to the VM.
 
+You can set an error policy for each disk. The error policy controls how the hypervisor behaves when an input/output error occurs when a disk is being written to or read. The default behavior stops the guest and generates a Kubernetes event.
+
+You can accept the default behavior, or you can set the error policy to one of the following options:
+
+* `report`, which reports the error in the guest.
+* `ignore`, which ignores the error. The Read or Write failure is undetected.
+* `enospace`, which produces an error indicating that there is not enough disk space.
```

Comment:
Might be good to have this list in "Configuring disk sharing by using LUN" also, so that users won't need to seek out the info in a different module. Maybe use a snippet?

Reply:
Users who configure disk sharing by using LUN are advised to set the error policy to report, not either of the other two options.